src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.cs

```diff
@@ -89,6 +90,14 @@ internal override CSharpSyntaxNode Root
             }
         }
 
+        internal bool ContainsNullCheckedParameter
```

Comment:
>ContainsNullCheckedParameter [](start = 22, length = 28)

If this is only used by `CSharpOperationFactory`, the checks (for `ParameterSymbol.IsNullChecked`) should probably be moved there.